### PR TITLE
Add image link to blogs create

### PR DESCRIPTION
**Pull request by hikes101**
We can decide on a proper default image later on, the current one is just a placeholder.

### CODE REVIEW

**Comment by hikes101** [comment]
[Add image input for blog create page](https://trello.com/c/3pNZNRpm/138-add-image-input-for-blog-create-page)

File: src/routes/blogs.js [hunk 1]
```diff
@@ -20,7 +27,7 @@ router
       date: moment().format("DD.MM.YYYY"),
       author: req.body.author,
       content: converter.convertPost(req.body.content),
-      imagelink: "/img/blogs/Platzhalter.jpg"
+      imagelink: imageInput(req.body.imagelink)
```
Comment (pheeria) [inline src/routes/blogs.js]: You could write the following way:
```js
imagelink: req.body.imagelink || "/img/blogs/default.jpg"
```

File: src/views/partials/blogs/create.hbs [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
   </div>
   <div class="form-group col-md-6">
-    <label>Content:</label>
-    <textarea name="content" class="form-control" rows="10"></textarea>
+    <label>Inhalt:</label>
+    <textarea name="content" class="form-control" rows="10" required></textarea>
+  </div>
+  <div class="form-group col-md-6">
+    <label>Link für das Titelbild:</label>
+    <input type="text" class="form-control" name="imagelink">
```
Comment (pheeria) [inline src/views/partials/blogs/create.hbs]: There exists `type="url"` :)

File: src/views/partials/blogs/create.hbs [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
     <input type="text" class="form-control" name="author" required>
   </div>
   <div class="form-group col-md-6">
-    <label>Content:</label>
-    <textarea name="content" class="form-control" rows="10"></textarea>
+    <label>Inhalt:</label>
+    <textarea name="content" class="form-control" rows="10" required></textarea>
+  </div>
+  <div class="form-group col-md-6">
+    <label>Link für das Titelbild:</label>
```
Comment (pheeria) [inline src/views/partials/blogs/create.hbs]: Could we move it before content?

Comment (hikes101) [reply src/views/partials/blogs/create.hbs]: Done

**Pull request merged**